composer.json

```diff
@@ -10,9 +10,9 @@
     "license": "MIT",
     "require": {
         "doctrine/dbal": "^3.3",
-        "illuminate/console": "^10.0",
-        "illuminate/filesystem": "^10.0",
-        "illuminate/support": "^10.0",
+        "illuminate/console": "^10.0|^11.0",
```

Comment:
single pipe (`|`) is only recommended for backwards compatibility in older versions of Composer.

>[!NOTE]
> A double pipe (`||`) will be treated as a **logical OR**.
>
> In _older_ versions of `Composer` the single pipe (`|`) was the recommended alternative to the **logical OR**.
>
> Thus for backwards compatibility the single pipe (`|`) will still be treated as a **logical OR**.
>

https://getcomposer.org/doc/articles/versions.md#version-range